packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -570,11 +570,12 @@ class FlutterPlugin implements Plugin<Project> {
                                         }
                                     }
                                 }
-                                schemes.each { scheme ->
-                                    hosts.each { host ->
-                                        if (!paths) {
-                                            appLinkSettings.deeplinks.add(new Deeplink(scheme: scheme, host: host, path: ".*", intentFilterCheck: intentFilterCheck))
-                                        } else {
+                                if(!hosts.isEmpty() || !paths.isEmpty()){
```

Comment:
do we not fallback if the customers only have schemes?

Reply:
yes,  the dev tool data table is listed by "domain view" "path view" "single url view" , if there is only schemes, we don't show it.

Reply:
I think for our current design we can't surface this case, but I think this is something we should go over again with stakeholders.

This is not something we need to figure in this PR though.